### PR TITLE
Add detection of EU users for Consent Modal

### PR DESCRIPTION
**Pull request by rogue-hacker**
For inside EU: 

image

For outside EU:

image

### CODE REVIEW

**Review by doylio — APPROVED** [review]

**Review by doylio — COMMENTED** [review]
It seems like this dialogue box will never appear to non-EU members, because they will automatically have "consentAcknowledged" set to true on boot

File: src/components/ConsentModal/ConsentModal.js [hunk 1]
```diff
@@ -19,11 +48,15 @@ function ConsentModal({ hasConsented, setConsented }) {
 
   useEffect(() => {
     const isBraveBrowser = navigator?.brave;
+    // Default opt-in for users outside the EU
+    if (!isInEU) {
+      localStorage.setItem("consentAcknowledged", "true");
+    }
```
Comment (dospore) [inline src/components/ConsentModal/ConsentModal.js:54]: I feel we cant default consent to true

File: src/components/ConsentModal/ConsentModal.js [hunk 2]
```diff
@@ -1,10 +1,39 @@
...
+      We use cookies on this site to enhance your user experience. The collection and storage of cookies on your device
+      to enhance your site experience and for analytical purposes. We want to respect your right to privacy, so you have
+      the option to select your cookie preferences. Note that changes to your preferences may impact your user
+      experience.{" "}
+      <a href="https://mycelium.xyz/privacy-policy/" rel="noopener noreferrer" target="_blank">
+        Privacy Policy
+      </a>
+      .
+    </>
+  );
```
Comment (dospore) [inline src/components/ConsentModal/ConsentModal.js:35]: Is this the same text?

**Review by dospore — CHANGES_REQUESTED** [review]
Requesting changes on the default opt in unless we add a way to opt out

**Pull request closed without merging**